editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -565,3 +582,100 @@ const SelectionAreaRectangle = React.memo(
 )
 
 SelectionAreaRectangle.displayName = 'SelectionAreaRectangle'
+
+const ElementsOutsideVisibleAreaIndicators = React.memo(
```

Comment:
please move this to a dedicated file

Reply:
https://github.com/concrete-utopia/utopia/pull/3842/commits/30a89e064c08ac48c0a1ebb9b506cdd91e68d927